config/configuration.yml.example

```diff
@@ -94,6 +94,12 @@
 #
 # === More configuration options
 #
+# Disable default module:
+#
+# By default user may choose which modules can be disabled
+# this can be done in lib/open_project/configuration.rb
+# disabled_modules:[list_of_modules_name]
```

Comment:
Yes and I think, as it is a yml document, we need to specify it as an array in yml format http://symfony.com/doc/current/components/yaml/yaml_format.html. The only thing I am unsure of is whether this works neatly with ENV-variables. Maybe @link or @machisuji know more about this?